Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9-slim
```

Comment:
Can we please use a newer version of python? are there any issues moving to 3.13?